dev-images.json

```diff
@@ -5,5 +5,6 @@
   "azure-cluster-api-controllers": "quay.io/ademicev/cluster-api-provider-azure:latest",
   "gcp-cluster-api-controllers": "quay.io/ademicev/cluster-api-provider-gcp:latest",
   "ibmcloud-cluster-api-controllers": "quay.io/kabhat/ibmcloud-cluster-api-controllers:dev",
+  "vsphere-cluster-api-controllers": "quay.io/dodvarka/cluster-api-provider-vsphere:latest",
```

Comment:
Any idea why all these images come from individuals repositories?

Reply:
These are development images.
This file can be passed to the operator when ran locally to instruct it to use custom images.
So people committed these in as they built them for their convenience.
I don't use it personally.

Reply:
We should probably remove that at some point

Reply:
Agree, in a separate PR.